docker-compose.yml

```diff
@@ -186,7 +186,7 @@ services:
       MYSQL_DATABASE: "silauth"
       MYSQL_USER: "silauth"
       MYSQL_PASSWORD: "silauth"
-      BASE_URL_PATH: "http://ssp-idp1.local/"
+      BASE_URL_PATH: "http://ssp-idp1.local/" # change this to "http://ssp-idp1.local:8085" for manual browser testing
```

Comment:
It might be worth making this a TODO or FIXME comment

Reply:
I think this is more documentary than needing to fix.

Reply:
@mtompset is right. However, there should be an action item to resolve the need to make this change. I'll file something in YouTrack